processor/src/main/java/org/mapstruct/ap/internal/processor/MapperCreationProcessor.java

```diff
@@ -307,7 +307,7 @@ private SortedSet<Type> getExtraImports(TypeElement element,  MapperOptions mapp
 
 
         for ( TypeMirror extraImport : mapperOptions.imports() ) {
-            Type type = typeFactory.getType( extraImport );
+            Type type = typeFactory.getAlwaysImportedType( extraImport );
```

Comment:
Would `explicit` be a better term? Or even `explicitImportByMappingAnnotation`..

Reply:
some javadoc otherwise..

Reply:
I named it `getAlwaysImportedType` because we might need to use it for something else in the future. You never know what can happen.

I'll add Javadoc though